test/controllers/homepages_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+describe HomepagesController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
```

Comment:
Looks like this was missed. In addition to the nominal case, there's an interesting edge case here when there are no works - how does the media spotlight section behave?